state/executor.go

```diff
@@ -150,17 +150,22 @@ func (e *BlockExecutor) ApplyBlock(ctx context.Context, state types.State, block
 	}
 
 	abciValUpdates := resp.EndBlock.ValidatorUpdates
-	err = validateValidatorUpdates(abciValUpdates, state.ConsensusParams.Validator)
+	protoState, err := state.ToProto()
```

Comment:
why do you need this change? `state.ConsensusParams.Validator` does not work?

Reply:
huh, my vs code complained about it when I changed it, but you are right now that I saw it, `state.Consensus.Validator` works